api/database/factories/PoolCandidateFilterFactory.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Classification;
+use App\Models\CmoAsset;
+use App\Models\OperationalRequirement;
+use App\Models\Pool;
+use App\Models\PoolCandidateFilter;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class PoolCandidateFilterFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = PoolCandidateFilter::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'has_diploma' => $this->faker->boolean(),
+            'has_disability' => $this->faker->boolean(),
+            'is_indigenous' => $this->faker->boolean(),
+            'is_visible_minority' => $this->faker->boolean(),
+            'is_woman' => $this->faker->boolean(),
+            'language_ability' => $this->faker->randomElement(['FRENCH', 'ENGLISH', 'BILINGUAL', null]),
+            'work_regions' => $this->faker->randomElements(
+                [
+                    'TELEWORK',
+                    'NATIONAL_CAPITAL',
+                    'ATLANTIC',
+                    'QUEBEC',
+                    'ONTARIO',
+                    'PRAIRIE',
+                    'BRITISH_COLUMBIA',
+                    'NORTH',
+                ],
+                3
+            ),
+        ];
+    }
+
+    public function configure()
+    {
+        return $this->afterCreating(function (PoolCandidateFilter $filter) {
+            $assets = CmoAsset::inRandomOrder()->limit(4)->get();
+            $classifications = Classification::inRandomOrder()->limit(3)->get();
+            $requirements = OperationalRequirement::inRandomOrder()->limit(2)->get();
+            $pools = Pool::inRandomOrder()->limit(2)->get();
```

Comment:
Connecting this to a single pool is more realistic, for now. Maybe if we could alternate between one and two that would be ideal, if easy.